src/amr/tagging/default_hybrid_tagger_strategy.h

```diff
@@ -28,49 +29,159 @@ void DefaultHybridTaggerStrategy<HybridModel>::tag(HybridModel& model,
 
     auto& N = model.state.ions.density();
 
+    double threshold = 0.1;
+
     // we loop on cell indexes for all qties regardless of their centering
-    auto start
+    auto start_x
         = layout.physicalStartIndex(PHARE::core::QtyCentering::dual, PHARE::core::Direction::X);
-    auto end = layout.physicalEndIndex(PHARE::core::QtyCentering::dual, PHARE::core::Direction::X);
+    auto end_x
+        = layout.physicalEndIndex(PHARE::core::QtyCentering::dual, PHARE::core::Direction::X);
 
-    auto endCell     = layout.nbrCells()[0] - 1;
-    double threshold = 0.05;
-    int idx          = 4;
-    double epsilon   = 0.1;
+    auto endCell_x = layout.nbrCells()[0] - 1;
 
+    // SAMRAI tags int* buffer is FORTRAN ordering so we set false to the view
+    auto tagsv
+        = core::NdArrayView<dimension, int, int*, /*c_ordering=*/false>(tags, layout.nbrCells());

```

Comment:
we could probably throw a 

```
template<std::size_t dim, typename DataType>
using FortranNdArrayView = NdArrayView<dim, DataType, DataType*, /*c_ordering=*/false>;
````

into ndarray_vector.h somewhere

Reply:
although maybe insufficient for ```DataType const*```

Reply:
:shrug:  ?